README.md

```diff
@@ -51,9 +51,9 @@ The optional `use_selection` boolean (defaults to `true`) controls whether to fo
 
 #### 2 Settings
 
-There is only one modifiable property in settings:
+There are some modifiable properties in settings:
 
-```js
+```json
```

Comment:
Do not modify the syntax of code blocks, comments are not supported in `js` code blocks.

Reply:
Ok, I'll revert it.